app/id_generation.py

```diff
@@ -0,0 +1,42 @@
+from fastapi import HTTPException
+import requests
+from settings import settings
+import random
+
+school_db_url = settings.db_url + "/school"
+
+
+class JNVIDGeneration:
+    def __init__(self, region, school, grade):
+        self.region = region
+        self.school = school
+        self.grade = grade
+        self.id = (
+            self.get_class_code()
+            + self.get_jnv_code()
+            + self.generate_three_digit_code()
+        )
+
+    def get_id(self):
+        return self.id
+
+    def get_class_code(self):
+        class_codes = {"9": "26", "10": "25", "11": "24", "12": "23"}
```

Comment:
we need to keep a track of this -- keeps on changing year after year :(

Reply:
actually, now that I think about it, I can probably automate generate it.